### ISSUE TITLE
Cannot access PVCs in namespaces in watchNamespace

### ISSUE THREAD

**Issue by tomcruz17** [mtougeron/k8s-aws-ebs-tagger#31]
**Describe the bug**
First of all, thank you for this! Works great with a minor issue I noticed when using the helm chart.

When setting `watchNamespace` to a namespace different from where the helm-generated manifests are, the tagger pods can't access the PVCs. For example, I deploy the resources in `ns1` and set `watchNamespace: "monitoring"`. This results in errors in the tagger pod logs:

```
k8s-aws-ebs-tagger-bdd94dd85-t8xqk k8s-aws-ebs-tagger E0930 01:04:41.988660       1 reflector.go:138] pkg/mod/k8s.io/client-go@v0.21.2/tools/cache/reflector.go:167: Failed to watch *v1.PersistentVolumeClaim: failed to list *v1.PersistentVolumeClaim: persistentvolumeclaims is forbidden: User "system:serviceaccount:ns1:k8s-aws-ebs-tagger" cannot list resource "persistentvolumeclaims" in API group "" in the namespace "monitoring"
```

I believe it just lacks `rolebindings` in the target namespaces in `watchNamespace`.

**To Reproduce**
1. Set `watchNamespace: monitoring` in chart values.
2. Deploy resources to `ns1`.
2. Print logs of k8s-aws-ebs-tagger pods and see the error above..

**Expected behavior**
k8s-aws-ebs-tagger pods should be able to access all PVCs in target namespaces in `watchNamespace`.

**Additional context**
I'm not a Go developer but it looks like this is purely helm chart-related which I'm familiar with. I could open a PR for this if needed.

**Comment by mtougeron** [comment]
Good catch. The chart doesn't setup the RBAC for anything but the one namespace. A PR would be appreciated but if you don't want to, I can get to this sometime next week.
